sdk/trace/tracetest/example_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package tracetest_test
+
+import (
+	"context"
+	"fmt"
+
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
```

Comment:
Why is it not simply?

```suggestion
	"go.opentelemetry.io/otel/sdk/trace"
```